load-testing-docs/load_testing.xml

```diff
@@ -0,0 +1,57 @@
+?xml version="1.0"?>
```

Comment:
this file isnt part of our load testing, and seems to hi cs291. Can we remove this file for the PR.